src/javascripts/components/forms/sectorsForm.js

```diff
@@ -0,0 +1,67 @@
+import sectorData from '../../helpers/data/sectorData';
+import systemsData from '../../helpers/data/systemsData';
+
+const getNewSector = () => ({
+  name: $('#sector-name').val() || false,
+  system_id: $('#system_id').val() || false
```

Comment:
Are you wanting to display the system associated with the sector? I'm confused as to why it is asked, but not displayed.

Reply:
System ids are part of the strech goal. But we can take it out if needed.

Reply:
Let us know what you would like to see.

Reply:
I agree with @brownabs stretch goals are for future development. Take it out.